compatibility_server/pip_checker.py

```diff
@@ -309,11 +354,15 @@ def _run_command(
             # TODO: Log the exception and monitor it after trying to decode
             # this into a requests.exception.* e.g. ReadTimeout. See:
             # http://docs.python-requests.org/en/master/_modules/requests/exceptions/
+            MMAP.measure_int_put(DOCKER_ERROR_MEASURE, 1)
+            MMAP.record(TMAP)
             raise PipCheckerError(
                 error_msg="An error occurred while running the command {} in"
                           "container. Error message: {}".format(command, e))
 
         if returncode and raise_on_failure:
+            MMAP.measure_int_put(DOCKER_ERROR_MEASURE, 1)
```

Comment:
This isn't necessarily a docker error.

Reply:
When https://github.com/GoogleCloudPlatform/cloud-opensource-python/pull/233 is merged, you can put this into the `if returncode > 128 ...` block

Reply:
done